source/contributing/contribution-guideline/index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: default
+title: Contribution Guideline 
+github_link: contributing/contribution-guideline/index.md
+indexed: true
+tags: [pullrequest, github, guideline, contribute, git, pull-request, fork]
+menu_title: Contribution Guideline
+menu_order: 20
+group: Contributing
+---
+
+<div class="toc-list"></div>
+
+## Introduction
+First of all, thank you! You have decided to contribute code to our software and become a member of the large shopware community. We appreciate your hard work and want to handle it with the most possible respect.
+
+To ensure the quality of our code and our products we have created a small guideline we all should endorse to. It helps you and us to collaborate with our software. Following these guidelines will help us to integrate your changes in our daily workflow. 
+
+## Requirements for a successful pull request
+To avoid that your pull request gets rejected, you should always check that you provided all necessary information, so that we can integrate your changes in our internal workflow very easily. Here is a check-list with some requirements you should always consider when committing new changes.
```

Comment:
maybe we should mention here, that they first should look up the existing PRs, so we don't get duplicates :smile: